### PR TITLE
launch app btn, modal with info

### PR DESCRIPTION
**Pull request by ankerx**

### CODE REVIEW

File: components/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      <Container>
+        <Card>
+          <CloseIconContainer>
+            <CloseIcon onClick={onClose} />
+          </CloseIconContainer>
+
+          <Box>
+            <Title>App is already live for investors and partners!</Title>
+            <Text>
+              Contact us at <Link href="mailto:support@0xkyc.id">support@0xkyc.id</Link> for credentials
```
Comment (dzienisz) [inline components/Modal.tsx]: Maybe instead of this we should put input for newsletter. Ask Adam about it.

File: components/Navbar.tsx [hunk 2]
```diff
@@ -73,9 +75,7 @@ export default function Navbar({ items }: NavbarProps) {
           </LogoWrapper>
         </NextLink>
         <NavItemList>
-          {items.map((singleItem) => (
-            <NavItem key={singleItem.href} {...singleItem} />
-          ))}
+          <CustomButton onClick={() => setIsModalOpened(true)}>Launch App</CustomButton>;
```
Comment (dzienisz) [inline components/Navbar.tsx]: Remove menu on mobile and show button instead of hamburger menu

File: components/NewsletterModal.tsx [hunk 3]
```diff
@@ -1,150 +0,0 @@
-import React, { useState } from 'react';
```
Comment (dzienisz) [inline components/NewsletterModal.tsx:1]: Why it's removed? Maybe we should use this?

File: pages/_app.tsx [hunk 4]
```diff
@@ -80,18 +82,18 @@ function MyApp({ Component, pageProps }: AppProps) {
...
   );
 }
 
 function Modals() {
-  const { isModalOpened, setIsModalOpened } = useNewsletterModalContext();
+  const { isModalOpened, setIsModalOpened } = useModalContext();
   if (!isModalOpened) {
     return null;
   }
-  return <NewsletterModal onClose={() => setIsModalOpened(false)} />;
```
Comment (dzienisz) [inline pages/_app.tsx:94]: I would line to see this modal maybe it's better?

File: pages/cookies-policy.tsx [hunk 5]
```diff
@@ -1,46 +0,0 @@
-import styled from 'styled-components';
```
Comment (dzienisz) [inline pages/cookies-policy.tsx:1]: We won't have cookie policy?

**Review by dzienisz — APPROVED** [review]
Talk with Adam or Dylan if they approve this

File: pages/cookies-policy.tsx (hunk 5 above)
Comment (ankerx) [reply pages/cookies-policy.tsx:1]: For now we don't have, can add this page later

**Pull request merged**